src/components/PostPageDetail/elements/Content.js

```diff
@@ -240,7 +240,7 @@ const Content = ({
               url: item.og.url,
               onHeaderPress: onPressDomain,
               onCardContentPress: navigateToLinkContextPage,
-              score: item.score,
+              score: item.credderScore,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `score` to `credderScore` improves the readability and maintainability of the code by making it clear that this specific score is related to Credder. However, please ensure that all references to `score` in other parts of the code have also been updated to `credderScore` to maintain consistency.
```diff
-              score: item.score,
+              credderScore: item.credderScore,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->